src/email/email.js

```diff
@@ -128,27 +146,13 @@ exports.startSMTP = function (config) {
         authOptional: true,
 
         onData: function (stream, session, callback) {
-            let subject, text;
+            let subject, text, from;
             const mailparser = new MailParser();
-            const fromAdd = ParseEmailAddress.parseOneAddress(session.envelope.mailFrom.address);
-            let receivedAddress = getRoomAliasFromEmailTo(session.envelope.rcptTo, config.bridge.domain);
-            let alias = "";
-            try {
-                if (receivedAddress[0] === "room") {
-                    alias = `#${receivedAddress[1]}:${receivedAddress[2]}`;
-                }
-                else if (receivedAddress[0] === "user") {
-                    // TODO: send message to user
-                    return;
-                }
-            }
-            catch (err) {
-                log.error(`Error resolving room address: ${err.message}`);
-            }
+            const fromAddress = ParseEmailAddress.parseOneAddress(session.envelope.mailFrom.address);
```

Comment:
I am loving the use of short utility functions btw, makes this a pleasure to review.